tests/test_adjointA.py

```diff
@@ -17,15 +17,15 @@
 @skipif_yask
 @pytest.mark.parametrize('mkey, shape, kernel, space_order, nbpml', [
     # 2D tests with varying time and space orders
-    ('layers', (60, 70), 'OT2', 4, 10), ('layers', (60, 70), 'OT2', 8, 10),
-    ('layers', (60, 70), 'OT2', 12, 10), ('layers', (60, 70), 'OT4', 4, 10),
-    ('layers', (60, 70), 'OT4', 8, 10), ('layers', (60, 70), 'OT4', 12, 10),
+    ('layers', (61, 71), 'OT2', 4, 20), ('layers', (61, 71), 'OT2', 8, 20),
```

Comment:
``61, 71``? This doesn't make any sense at all

Reply:
This will be reverted, I'm seting it up a way I like to tests and find a way to make it not "crash". CUrrently the sparse FD is fucking up everything as it generates way too many expressions, like for space_order 8 it's 100 in 2D and 200 in 3D.